src/shared/components/community/sidebar.tsx

```diff
@@ -251,7 +251,7 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
         <li className="list-inline-item">
           <Link
             className="badge badge-primary"
-            to={`/modlog/community/${this.props.community_view.community.id}`}
+            to={`/modlog/${this.props.community_view.community.id}`}
```

Comment:
Oops, my bad. Thanks for catching.

Reply:
TBH I think I might have introduced that bug. That route was correct until the query param change I made.